workflows/src/main/kotlin/com/template/flows/ApproveAppointmentRequest.kt

```diff
@@ -31,7 +36,10 @@ import java.util.*
 @InitiatingFlow
 @StartableByRPC
 class ApproveAppointmentRequest(private val alice: Party,
```

Comment:
What if we add a third patient? What do you think about providing a list of patients instead of hardcoding parties for alice and bob?